addons/website_event_track/static/src/js/website_event_track.js

```diff
@@ -9,6 +9,67 @@ publicWidget.registry.websiteEventTrack = publicWidget.Widget.extend({
         'input #event_track_search': '_onEventTrackSearchInput',
     },
 
+    /**
+     * @override
+     */
+    start: function () {
```

Comment:
I don't remember all this code from my previous review.
Isn't it possible to let bootstrap handle the show/hide of the popover?

I feel like it has worked without all that JS code at some point :thinking: